screens/AllUsersListScreen/HeaderList.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React, {useContext} from 'react';
+import {
+  Image,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import ChatIcon from '../../assets/icons/Chat.svg';
+import {theme} from '../../assets/theme/theme';
+import {GlobalDataContext} from '../../contexts/context';
+import Search from '../../components/Search/Search';
+import styles from './style';
+
+
+const HeaderList = ({data, handleSearch, setEmptyDataMessage, navigation}) => {
+    const { imageUrl } = useContext(GlobalDataContext);
+
+    // console.log('Header List component rendered >>>>>>');
```

Comment:
?

Reply:
This component is a header, I just misspelled the component name here, which I already changed and removed the console )